tests/modules/programs/neovim/extra-lua-init.nix

```diff
@@ -0,0 +1,21 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  config = {
+    programs.neovim = {
+      enable = true;
+
+      extraLuaConfig = ''
+        -- extraLuaConfig
+      '';
+    };
+    nmt.script = ''
+      nvimFolder="home-files/.config/nvim"
+      assertFileContent "$nvimFolder/init.lua" ${
+        pkgs.writeText "init.lua-expected" "-- extraLuaConfig "
```

Comment:
```suggestion
        pkgs.writeText "init.lua-expected" ''
          -- extraLuaConfig
        '';
```

Reply:
Thanks for spotting that, I got ahead of myself 😅

Reply:
Fixed in 6ee2f04